kokiri/src/core/game.cpp

```diff
@@ -0,0 +1,48 @@
+#include "kkr.hpp"
+#include "game.hpp"
+
+#include <SDL2/SDL.h>
+
+namespace Kokiri {
+    namespace Core {
```

Comment:
What core means? It seems a too broad namespace name @rafaelcn

Reply:
Could be base and it would work as well, everything that's not too specific I'm gonna place it under Core. I don't see a reason why should I create more namespaces for these files yet